tests/fixtures/model_form_fields_all.py

```diff
@@ -13,4 +13,4 @@ class User4(ModelForm):
 
     class Meta:
         model = User2
-        fields = b'__all__'
\ No newline at end of file
+        fields = '__all__'
```

Comment:
@avallbona please leave this as it was since this is to cover a case in which the attribute could be a bytes literal, and we don't want the checkers to fail in this case.

Reply:
done